interface/src/Application.cpp

```diff
@@ -715,6 +716,7 @@ static const QString STATE_PLATFORM_ANDROID = "PlatformAndroid";
 static const QString STATE_LEFT_HAND_DOMINANT = "LeftHandDominant";
 static const QString STATE_RIGHT_HAND_DOMINANT = "RightHandDominant";
 static const QString STATE_STRAFE_ENABLED = "StrafeEnabled";
+static const QString STATE_CAMERA_CAPTURE_MOUSE = "CaptureMouse";
```

Comment:
It’s not quite a camera mode (in that you can be in first person or third person or otherwise and have the mouse captured or not) but it does affect the camera...I will change to STATE_CAPTURE_MOUSE